type_templates/scn.js

```diff
@@ -1,8 +1,10 @@
 import * as THREE from 'three';
 
 import metaversefile from 'metaversefile';
+import {VRMSpringBoneImporter} from '@pixiv/three-vrm/lib/three-vrm.module.js';

```

Comment:
We need to go through the `metaversefile` api for this. `VRMSpringBoneImporter` is an internal thing that should not be imported like this by user apps (which is what the loaders are).